embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testcases/session/BackgroundActivityDisabledTest.kt

```diff
@@ -45,19 +42,14 @@ import org.junit.runner.RunWith
 @RunWith(AndroidJUnit4::class)
 internal class BackgroundActivityDisabledTest {
 
+    private lateinit var executor: BlockingScheduledExecutorService
+
     @Rule
     @JvmField
     val testRule: SdkIntegrationTestRule = SdkIntegrationTestRule {
-        val clock = FakeClock()
-        val initModule = FakeInitModule(clock, createForIntegrationTest())
-        val workerThreadModule =
-            FakeWorkerThreadModule(initModule, Worker.Background.LogMessageWorker)
-
-        EmbraceSetupInterface(
-            overriddenClock = clock,
-            overriddenInitModule = initModule,
-            overriddenWorkerThreadModule = workerThreadModule,
-        )
+        EmbraceSetupInterface(workerToFake = Worker.Background.LogMessageWorker).also {
+            executor = it.getFakedWorkerExecutor()
+        }
```

Comment:
Similar feedback to FlutterInternalInterfaceTest - I don't think this is required anymore if we refactor the assertions a little bit